cpp/tests/utilities/column_utilities.cuh

```diff
@@ -40,5 +42,19 @@ void expect_columns_equal(cudf::column_view lhs, cudf::column_view rhs);
 void expect_equal_buffers(void const* lhs, void const* rhs,
                           std::size_t size_bytes);
 
+/**---------------------------------------------------------------------------*
+ * @brief Print a column for debugging.
+ *
+ * @param col The column to print
+ *---------------------------------------------------------------------------**/
+template <typename Element>
+void print_column(cudf::mutable_column_view col) {
```

Comment:
Printing shouldn't need to change the column, yes? 
```suggestion
void print_column(cudf::column_view col) {
```

Reply:
It doesn't need to change the column, but since the column_view's `data()` and `null_mask()` return `const *`, we get this compiler error attempting to `reinterpret_cast` the null mask:
```
../tests/utilities/column_utilities.cuh(54): error: reinterpret_cast cannot cast away const or other type qualifiers
```

Reply:
Then do this instead:
```
    reinterpret_cast<gdf_valid_type const*>(col.null_mask()),
```

Reply:
We'll need a new print function that doesn't use this old one anyways.

Reply:
It compiles if I do a c-style cast ala `(gdf_valid_type*) col.null_mask()`, but it feels dirty.

Reply:
Yeah, don't do that. Just add the `const` like I said.

Reply:
Done!